packages/ciphernode/scripts/test.sh

```
@@ -0,0 +1,7 @@
+#!/usr/bin/env bash
+
+set -e
+
+pushd ./evm && ./scripts/build_fixtures.sh && popd
+
+cargo test -- $@
```

Shellcheck

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)